addons/project/static/src/components/project_task_name_with_subtask_count_char_field/project_task_name_with_subtask_count_char_field.js

```diff
@@ -6,7 +6,7 @@ import { formatChar } from '@web/views/fields/formatters';
 
 class ProjectTaskNameWithSubtaskCountCharField extends CharField {
     get formattedSubtaskCount() {
-        return formatChar(this.props.record.data.allow_subtasks && this.props.record.data.child_text || '');
+        return this.props.record.data.subtask_count ? `(${this.props.record.data.closed_subtask_count}/${this.props.record.data.subtask_count} subtasks)` : '';
```

Comment:
@RaoufGhrissi Don't wanna break lines? Don't want to use double quotes? :sweat_smile:

Reply:
haha no @auon-odoo i didn't ignore your comments (we don't do that here), i just pushed yesterday to let xavier see the story of the compiler selector, i will finish it today